Pilot/pilotTools.py

```diff
@@ -488,8 +489,9 @@ def __init__( self ):
     self.cmdOpts = ( ( 'b', 'build', 'Force local compilation' ),
                      ( 'd', 'debug', 'Set debug flag' ),
                      ( 'e:', 'extraPackages=', 'Extra packages to install (comma separated)' ),
-                     ( 'E:', 'commandExtensions=', 'Python module with extra commands' ),
-                     ( 'X:', 'commands=', 'Pilot commands to execute commands' ),
+                     ( 'E:', 'commandExtensions=', 'Python modules with extra commands' ),
+                     ( 'X:', 'commands=', 'Pilot commands to execute' ),
+                     ( 'Z:', 'commandOptions=', 'Options parsed by command modules' ),
```

Comment:
TBH I am a bit tempted to remove the "1 letter switch" and leave only the long one.